ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="selectable-card is-rounded no-flex" data-test-selectable-card-container={{@cardTitle}}>
+  <div class="is-flex-between is-fullwidth card-details" data-test-selectable-card={{@cardTitle}}>
+    <h3 class="title is-5">{{@cardTitle}}</h3>
+    <LinkTo
+      class="has-icon-right is-ghost is-no-underline has-text-semibold"
+      @route={{@actionTo}}
+      @query={{hash itemType=@queryParam}}
+      data-test-action-text={{@actionText}}
+    >
+      {{@actionText}}
+      {{#if @actionText}}
```

Comment:
I would expect that we don't render a link at all if `!@actionText` -- maybe we move this to wrap the `LinkTo`?